### ISSUE TITLE
Pass attribute mask for chemberta model

### ISSUE THREAD

**Issue by arunppsg** [deepchem/deepchem#3597]
Since we are padding tokens for masked language model training here,
https://github.com/deepchem/deepchem/blob/e815ffb5191520e078678b3183546c6042413507/deepchem/models/torch_models/hf_models.py#L219-L220
we should pass `attention_mask` to the model.

Steps to reproduce warning from `transformers` library:
```py
from deepchem.models import Chemberta
imort deepchem as dc

task, dataset, transformer = dc.molnet.load_delaney(featurizer=dc.feat.DummyFeaturizer(), reload=True)
train, test, valid = dataset

model = Chemberta(task='mlm', model_dir='pretrain-dir')
model.fit(train, nb_epoch=1, checkpoint_interval=1)
```
It will show a warning as
```
We strongly recommend passing in an `attention_mask` since your input_ids may be padded. See https://huggingface.co/docs/transformers/troubleshooting#incorrect-output-when-padding-tokens-arent-masked.
```
